src/types/annotator.ts

```diff
@@ -161,6 +161,26 @@ export type SidebarLayout = {
   toolbarWidth: number;
 };
 
+export type Rect = {
+  type: 'rect';
+  left: number;
+  top: number;
+  right: number;
+  bottom: number;
+};
+
+export type Point = {
+  type: 'point';
+  x: number;
+  y: number;
+};
+
+/**
+ * Shapes used to define 2D regions of a document that an annotation can be
+ * associated with.
+ */
+export type Shape = Rect | Point;
```

Comment:
We could perhaps have used the `DOMPoint` and `DOMRect` types here, but these plain types can be more easily extended to include additional shapes such as a polygon or line in future. They can also be sent between frames, although I'm not sure if we'll need that yet. The downside is that you don't get the `DOMRect` utility methods.